### PR TITLE
Bug 2091638: add yaml tab to blockpool

### PR DESCRIPTION
**Pull request by weirdwiz**
Screenshot from 2022-06-06 11-57-34
Screenshot from 2022-06-06 12-57-49

### CODE REVIEW

File: packages/ocs/block-pool/BlockPoolDetailsPage.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { referenceForModel } from '@odf/shared/utils';
 import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
 import { useTranslation } from 'react-i18next';
 import { RouteComponentProps, useLocation } from 'react-router-dom';
+import { YAMLEditorWrapped } from '../../odf/components/resource-pages/CommonDetails';
```
Comment (SanjalKatiyar) [inline packages/ocs/block-pool/BlockPoolDetailsPage.tsx:10]: don't import from odf package

Comment (SanjalKatiyar) [reply packages/ocs/block-pool/BlockPoolDetailsPage.tsx:10]: shift wrapped components to shared... then import from there...

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**